### ISSUE TITLE
paradox vacuum doesn't suck bricks

### ISSUE THREAD

**Issue by jonnylegoboy911** [UchuServer/Uchu#80]
AG mission for Wisp Lee

**Comment by MickVermeulen** [comment]
This is weird, though, did it help reloading into the world? Did the problem persist?

**Comment by jonnylegoboy911** [comment]
Well of the two times I've tried it, it seems to allow me to suck up one brick (not sure if it adds to the tally but either way it's 0/8 when you reload) then all the bricks are missing when I reload. So I can't really tell.

**Comment by jonnylegoboy911** [comment]
![bricks](https://user-images.githubusercontent.com/71595434/94995320-72401380-0595-11eb-9dc0-d793fbb6b024.png)
and it seems a couple of reloads later they return but the mission is complete

**Comment by TheMatt2** [comment]
I was able to pick up exactly one brick. The one furtherst right in the picture.
However, I could not pickup any others.
Restarting the server allowed me to collect that one brick repeatidly, which may be its own issue.

**Comment by TheNexusAvenger** [comment]
Something I noticed is that after sucking up a brick, smashing becomes ineffective despite all packets being received and the game tick going. Oddly enough, then running to a Mech and having it fire resulted in the server being stuck at "Can't keep up, skipped 20/20 ticks!", which I was able to reproduce 4 times. Haven't looked into the code, but I am wondering if there is some sort of resource lock that isn't being released.
